conference.js

```diff
@@ -2699,6 +2699,13 @@ export default {
     onProxyConnectionEvent(event) {
         if (!this._proxyConnection) {
             this._proxyConnection = new JitsiMeetJS.ProxyConnectionService({
+
+                /**
+                 * Pass the {@code JitsiConnection} instance which will be used
+                 * to fetch TURN credentials.
+                 */
+                jitsiConnection: APP.connection,
```

Comment:
Do you think we should error or no-op something if there is no APP.connection? Maybe that kind of error handling is a much bigger, separate task, though.

Reply:
`APP && APP.connection` ?

Reply:
In LJM it is optional, so it won't crash. Only this part can

Reply:
I was thinking if there is not connection somehow that it would default to using stun only, which maybe is okay too or maybe isn't. Like do we want to make sure for jitsi-meet that we always pass in APP.connection? I'm guessing right now it doesn't matter.

Reply:
so is it that you just want to have an error logged for easier debugging ? because I guess crashing is not a good idea

Reply:
A log line in LJM would be handling more places I guess. We can log "hey there's jitsiconnection so we'll get TURN" or "no JitsiConnection so there will be no TURN for you today"